### ISSUE TITLE
Can't compile example projects on windows 10

### ISSUE THREAD

**Issue by alexzlov** [dylan-lang/opendylan#1117]
I've installed opendylan 2014.1 release from opendylan.org and set up pellec c distro and environment variables (lib, include etc). When I try to build one of example projects provided (hello-world and opengl stuff) I got the following link errors:

`building targets: exe
LinkDLL C:\Users\Alex\AppData\Roaming\Open-Dylan\bin\DxCMNDYL.dll C:\Users\Alex\AppData\Roaming\Open-Dylan\lib\DxCMNDYL.lib : C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\_glue.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\library.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\macros.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\common-extensions.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\win32-common-extensions.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\locators-protocol.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\streams-protocol.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\random.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\numerics.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\format.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\byte-vector.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\timers.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\transcendentals.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\12=utilities.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\13=machine-word.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\14=logicals.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\15=arithmetic.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\16=division.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\17=shift.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\18=unsigned.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\19=signal-overflow.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\20=double.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\21=unsigned-double.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\timer_helpers.obj C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\version.res
POLINK: 
warning: Invalid machine type in object 'C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\timer_helpers.obj'.
POLINK: warning: No subsystem specified; assuming CONSOLE.

POLINK: 
error: Unresolved external symbol '_timer_get_point_in_time' - referenced from 'C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\timers.obj'.
POLINK: error: Unresolved external symbol '__imp_QueryPerformanceCounter' - referenced from 'C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\timer_helpers.obj'.
POLINK: error: Unresolved external symbol '__imp_QueryPerformanceFrequency' - referenced from 'C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\timer_helpers.obj'.
POLINK: fatal error: 3 unresolved external(s).

Building target DxCMNDYL.dll: "polink /DLL /OUT:"C:\Users\Alex\AppData\Roaming\Open-Dylan\bin\DxCMNDYL.dll" /IMPLIB:"C:\Users\Alex\AppData\Roaming\Open-Dylan\lib\DxCMNDYL.lib" /LIBPATH:"C:\Users\Alex\AppData\Roaming\Open-Dylan\lib" /LIBPATH:"D:\Soft\opendylan\lib" /NODEFAULTLIB /DEBUG /DEBUGTYPE:CV /BASE:0x66DC0000 /VERSION:0.0 /ENTRY:common_dylanDll@12 @"C:\Users\Alex\AppData\Local\Temp\jamE3QLF.tmp" @"C:\Users\Alex\AppData\Local\Temp\jamBKJAM.tmp" kernel32.lib dylan-support.lib crtmt.lib" failed with return status 1`

Is it a trouble with my environment or something?
Thanks in advance.

**Comment by pedro-w** [comment]
I can't test it just at the minute but I have had OpenDylan 2014.1 working on Windows 10 quite recently (in the last month)
I used the 32-bit version of Pelles C *8.00* - I think @cgay reported there were problems with version 9 (which hasn't been released yet) http://colabti.org/irclogger/irclogger_log/dylan?date=2018-06-15#l2
You can get version 8 from the German mirror of Pelles site. http://www.pellesc.de/index.php?page=download&lang=en

**Comment by alexzlov** [comment]
@pedro-w thanks for the information. I tried Pelles C ver. 8.0 and got the same results as with 9.0 version (unresolved externals in timer_helpers.obj). The weirdest thing is that Opendylan ver. 1.0 beta 1 works flawlessly with Pelles C ver. 9.0 (and 8.0 too) and vc6.0 backends. Actually, it's the only version that works correctly on my windows 10 machine, without any errors (all the example projects work and interactive listener too) and I was able to finish all the tutorials with Opendylan 1.0 beta 1.

**Comment by pedro-w** [comment]
I see you've got 
```
warning: Invalid machine type in object 'C:\Users\Alex\AppData\Roaming\Open-Dylan\build\common-dylan\timer_helpers.obj'.
```
which probably  the root cause of subsequent errors.

Is there a chance you have some old or corrupted files there? Otherwise I can't see what you and I are doing differently.

**Comment by alexzlov** [comment]
Got some progress. I wiped out all the records mentioning Opendylan from the registry, erased everything in C:/Users/User_Name/AppData/Roaming/Open-Dylan... and performed a clean install of 2014.1 version. After that, I was able to compile an OpenGL sample and the REPL. But, when I tried to evaluate something in the REPL, I got following error message:
`Error: Reference to undefined binding "format-to-string".`  After that, the REPL froze. It seems that the playground wants to output something for me using that unavailable method. 
So, your workaround worked, @pedro-w, thank you. But after all, it seems to me that Opendylan 1.0 beta 1 is the most stable and usable version.
